docs/source/getting_started/installation.rst

```diff
@@ -98,6 +98,32 @@ Usage with JupyterHub
 Read the details on our :ref:`JupyterLab on JupyterHub documentation page <jupyterhub>`.
 
 
+Usage with Jupyverse
+--------------------
+
+`Jupyverse <https://github.com/jupyter-server/jupyverse/>`__ is a next-generation Jupyter server based on
+`FastAPI <https://fastapi.tiangolo.com/>`__. It can be used instead of
+`jupyter-server <https://github.com/jupyter-server/jupyter_server/>`__, the Jupyter server installed by default with JupyterLab.
+Note that ``jupyter-server`` extensions won't work with ``jupyverse`` (for which there might be equivalent plugins).
+
+You can install ``jupyverse`` with ``pip``:
+
+.. code:: bash
+
+    pip install jupyverse[auth, jupyterlab]
+
+or with ``mamba``:
```

Comment:
Should there also be a mention of `conda`, since the rest of the document mentions it?

Reply:
Done, thanks.